api/clients/eigenda_client.go

```diff
@@ -115,30 +115,29 @@ func NewEigenDAClient(log log.Logger, config EigenDAClientConfig) (*EigenDAClien
 		return nil, fmt.Errorf("new disperser-client: %w", err)
 	}
 
-	lowLevelCodec, err := codecs.BlobEncodingVersionToCodec(config.PutBlobEncodingVersion)
-	if err != nil {
-		return nil, fmt.Errorf("create low level codec: %w", err)
-	}
-
-	var codec codecs.BlobCodec
+	var polynomialForm codecs.PolynomialForm
 	if config.DisablePointVerificationMode {
```

Comment:
Might be a good time to update this weirdly named flag to something more explicit like the polynomial form being passed directly?

Feel like the fact that coeff form has point verification feature is VERY context dependent on understanding the way the eigenda backend is doing things etc. This should prob just be documented in the ENUM and flag, and then users can choose which form they want to use accordingly?